#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "teraranger_mutliflex_cfg"

gen = ParameterGenerator()

gen.add("Sensor_0", bool_t, 1, "Activate sensor", True)
gen.add("Sensor_1", bool_t, 1, "Activate sensor", True)
gen.add("Sensor_2", bool_t, 1, "Activate sensor", True)
gen.add("Sensor_3", bool_t, 1, "Activate sensor", True)
gen.add("Sensor_4", bool_t, 1, "Activate sensor", True)
gen.add("Sensor_5", bool_t, 1, "Activate sensor", True)
gen.add("Sensor_6", bool_t, 1, "Activate sensor", True)
gen.add("Sensor_7", bool_t, 1, "Activate sensor", True)

exit(gen.generate(PACKAGE, "teraranger_mutliflex_cfg", "TerarangerHubMultiflex"))
